aries-site/src/pages/templates/drill-down-navigation.mdx

```diff
@@ -1,3 +1,50 @@
-import { ComingSoon } from '../../components';
+import { DiagramPreview, Example } from '../../layouts';
+import { DrillDownCardExample, DrillDownDataTableExample, DrillDownListExample } from '../../examples';
 
-<ComingSoon />
\ No newline at end of file
+## When to Use:
```

Comment:
Ahead of this section, I would expect an introduction explaining the purpose of Drill-down navigation and why I would use it. For example:
"Drill-down navigation keeps the users attention focussed on the right level of detail at the right time. Progressively revealing more granular detail allows the UI to remain clean and focussed, while removing extraneous information which can lead to cognitive overload..."

Reply:
Are we saying "drill down navigation" is the same as "hub and spoke"? Are we intentionally avoiding that term? Should we allude to "also known as"? Or are we seeing "drill down" + "ascending" as navigation means to traverse a hub & spoke content architecture?

Reply:
Do you feel that the explanation before getting into "When to Use" doesn't explicitly convey what Drill Down Navigation is? Just trying to further understand. I can a bit more granularity.

Reply:
Drill Down Navigation is Hub&Spoke. I was avoiding the term as many people see it as a type of page layout and I feel "Drill Down" is more aligned with the rest of the type of Navigation conventions.

Reply:
> Ahead of this section, I would expect an introduction explaining the purpose of Drill-down navigation and why I would use it. For example: "Drill-down navigation keeps the users attention focussed on the right level of detail at the right time. Progressively revealing more granular detail allows the UI to remain clean and focussed, while removing extraneous information which can lead to cognitive overload..."

I've reworded with some of your wording below:

Allows users to progressively descend 'down' branches of a hierarchical tree into a child 
screen to drill down into more granular details. Drill Down Navigation allows for a consistent
and focused experience that helps eliminate cognitive overload.